var/spack/repos/builtin/packages/synapsetool/package.py

```diff
@@ -31,11 +31,13 @@ class Synapsetool(CMakePackage):
        connectivity data with billions of connections."""
     homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/hpc/synapse-tool"
     url      = "ssh://bbpcode.epfl.ch/hpc/synapse-tool"
+    git      = "ssh://bbpcode.epfl.ch/hpc/synapse-tool"
 
-    version('develop', git=url, submodules=True)
-    version('0.4.1', git=url, tag='v0.4.1', submodules=True, preferred=True)
-    version('0.3.3', git=url, tag='v0.3.3', submodules=True)
-    version('0.2.5', git=url, tag='v0.2.5', submodules=True)
+    version('develop', submodules=True)
+    version('0.5.0', tag='v0.5.1', submodules=True)
```

Comment:
I guess v0.5.1 is a typo, shouldn't it be v0.5.0?

Reply:
Correct!